com.adrienplazas.Metronome.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.adrienplazas.Metronome",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "metronome",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
+        "test-args": [
+            "--socket=x11",
+            "--share=network"
+        ],
```

Comment:
there is no network on the builder. so if expect to work (like for cargo), it won't.

Reply:
and there is no X11 either. So 'run-tests' will like fail if you expect this